static/bidder-info/boldwin.yaml

```diff
@@ -0,0 +1,14 @@
+maintainer:
+  email: "wls_demo_box@smartyads.com"
```

Comment:
"smatryads.com" - we already have it in Smartyads adapter. Please confirm this email here is correct.

Reply:
this email is correct